### ISSUE TITLE
Data augmentation for part segmentation?

### ISSUE THREAD

**Issue by ThibaultGROUEIX** [charlesq34/pointnet#140]
Hi, awesome work and great repo ! I am trying to reimplement it in pytorch and get the paper's result of 83.7 mean IOU for part segmentation. 

To do so, I have two questions on part segmentation :
1)- Do you perform data augmentation for part segmentation on Shapenet16 ? I couldn't find the info in the paper or the code.

If yes: could you say which kind ?
If no: why do it for classification on modelnet40 and not on shapenet16 ?

2)- What is the difference in performance with and without the T-NET ?

Thanks in advance for your help,
Best regards,
Thibault

**Comment by guobaisong** [comment]
> It seems that you do not do data-augmentation indeed, from your calls to provider.py in train.py. Do you have insight on why ?
> Can you confirm that no further preprocessing than normalization is done ?



> Hi, awesome work and great repo ! I am trying to reimplement it in pytorch and get the paper's result of 83.7 mean IOU for part segmentation.
> 
> To do so, I have two questions on part segmentation :
> 1)- Do you perform data augmentation for part segmentation on Shapenet16 ? I couldn't find the info in the paper or the code.
> 
> If yes: could you say which kind ?
> If no: why do it for classification on modelnet40 and not on shapenet16 ?
> 
> 2)- What is the difference in performance with and without the T-NET ?
> 
> Thanks in advance for your help,
> Best regards,
> Thibault

Hi, can you get the accuracy in paper without data augment?
